processor/samplingprocessor/probabilisticsamplerprocessor/testdata/config.yaml

```diff
@@ -2,8 +2,21 @@ receivers:
   examplereceiver:
 
 processors:
+  # The probabilistic_sampler sets trace sampling by hashing the trace id of
+  # each span and making the sampling decision based on the hashed value. It
+  # also obeys the "sampling.priority" semantic convention as defined by
+  # OpenTracing. See
+  # https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table
   probabilistic_sampler:
+    # the percentage rate at which traces are going to be sampled. Defaults to
+    # zero, i.e.: no sample. Values greater or equal 100 are treated as
+    # "sample all traces".
     sampling_percentage: 15.3
+    # hash_seed allows one to configure the hashing seed. This is important in
+    # scenarios where multiple layers of collectors have different sampling
+    # rates: if all layers use the same seed, all data passing one layer may
+    # pass the next even if they have different sampling rates. Having different
+    # seeds at different layers avoid that problem.
```

Comment:
This description could use some clarification. I had to think too deeply about the scenario this addresses. The "different sampling rates" bit is a red herring - what matters is whether the user expects the sampling rates in different layers to be multiplied, so 10% x 10% = 1%, for example.